src/depcache.erl

```diff
@@ -812,3 +840,6 @@ flush_message(Msg) ->
     after 0 ->
             ok
     end.
+
+callback(Type, Name, {M, F, A}) ->
+    spawn(M, F, [{Type, Name} | A]).
```

Comment:
If you would use `apply`, please wrap it with try/catch otherwise the callback could crash the entire depcache process.